src/Microsoft.AspNet.WebHooks.Receivers/WebHooks/WebHookReceiver.cs

```diff
@@ -13,16 +13,14 @@
 using System.Threading.Tasks;
 using System.Web.Http;
 using System.Web.Http.Controllers;
-using System.Web.Http.Dependencies;
```

Comment:
If a changed line in the ASP.NET code doesn't mention "form data", VS changed it when I accepted its suggestions or when I cleaned up `using`s afterward.